functions.php

```diff
@@ -741,6 +746,8 @@ function export_graphs(&$export, $export_path) {
 					}
 				}
 			}
+
+			export_debug('There are ' . sizeof($ngraph) . ' total graphs to export for all trees');
```

Comment:
I'm not sure if `$ngraph` is the correct variable name?

Reply:
$ngraph array was added to if is_graph_allowed on line 743-744.  I think this is correct.

Reply:
@thrca, can you change the "sizeof(" here to "cacti_sizeof("?  That will complete the requested changes.

Reply:
stet that, I still see soft-tabs.  We use a standard of hard tabs with a tab size of 4 chars.

```
set ts=4
```

Reply:
It looks like tabs are now being used but agree the sizeof should really be cacti_sizeof now.